configuration/pih/pih-config-liberia-harper-jjdossen.json

```diff
@@ -10,7 +10,21 @@
     "systemAdministration",
     "myAccount",
     "overviewReports",
-    "dataExports"
+    "dataExports",
+    "checkInHomepageApp",
+    "vitals",
+    "allergies",
+    "relationships",
+    "programs",
+    "mentalHealth",
+    "patientDocuments",
+    "conditionList",
+    "primaryCare",
+    "managePrinters",
+    "bmiOnClinicianDashboard",
+    "activeVisits",
+    "waitingForConsult",
+    "relationships"
   ],
```

Comment:
Just as an fyi, this could all be moved up to "pih-config-liberia-harper" if we want the same components turned on for Kouka, Demo (not sure what server that is) and the JJ dossen production server.

Reply:
Yes except "managePrinters". We can still have it on kouka though won't be used there.

Reply:
Sound good!